### ISSUE TITLE
Does MultiBit remember "old" public/private key pairs?

### ISSUE THREAD

**Issue by DavidKDeutsch** [Multibit-Legacy/multibit#248]
Hello,

 Let's say I have a wallet that contains 4 bitcoins under public key "A", and I make a transaction of one bitcoin to someone else who owns public key "B". What I see in the transaction details is 

A-->B 1 bitcoin
A-->C 3 bitcoins

OK, so now I own key "C," and "A" no longer has any bitcoins. If someone then transfers money to "A," will MultiBit recognize that, or is it "done" with key "A" since it does not have a balance anymore?

In a nutshell, my scenario is that the owner of key "B" will be sending the money back to "A," and I just want to make sure I can get it. Thanks.
